distributed/tensor_parallelism/example.py

```diff
@@ -48,17 +51,47 @@
 to use and our `parallelize_module` API will parse and parallelize the modules
 based on the given `ParallelStyle`. We are using this PyTorch native Tensor
 Parallelism APIs in this example to show users how to use them.
+
+Also one can specify the flag `run_seq_parallel` to run the example about
```

Comment:
I think we should have a separate example for sequence parallel, maybe this file can be renamed to `tensor_parallel_example.py` where it tests TP (maybe with 2-D together), and we should have a separate example that showcase sequence parallel, so that we could have more explanation about SP, wdyt?

Reply:
There are so many duplicate code so I didn't split them into different files.